openupgradelib/openupgrade.py

```diff
@@ -811,8 +811,8 @@ def map_values(
     mapped
     :param target_column: the database column, or model field (if 'write' is \
     'orm') that the new values are written to
-    :para mapping: list of tuples [(old value, new value)]
-    Old value True represents "is set", False "is not set".
+    :param mapping: list of tuples [(old value, new value)]
+        Old value True represents "is set", False "is not set".
```

Comment:
Are you sure this space goes here? The rest of the params that have multiple lines don't indent this way